### PR TITLE
CI: Update CEF builds

### PR DESCRIPTION
**Pull request by RytoEX**
### Description

CI: Update CEF builds

Update CEF builds to patch two CVEs:
 * CVE-2023-4863
 * CVE-2023-5217

### Motivation and Context

Want to address CVEs.

### How Has This Been Tested?

I haven't tested this.

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [ ] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

File: buildspec.json [hunk 1]
```diff
@@ -28,15 +28,18 @@
...
+                "linux-aarch64": "c07ada8e18633308771d592d0313c6fb17d5805448e8921ebc51afb19e0e429e",
+                "windows-x64": "dfb15d26daa1536cd5be10f3ad56a5888cacf226099165ec6ab70b60111b6184"
             },
             "revision": {
-                "macos-x86_64": 2,
-                "macos-arm64": 2
+                "macos-x86_64": 3,
+                "macos-arm64": 3,
+                "linux-x86_64": 2,
+                "linux-arm64": 2,
```
[the remark this answers is not in the document]
Comment (RytoEX) [reply buildspec.json:41]: It was, and I fixed it in force-push just before this.
https://github.com/obsproject/obs-studio/compare/a8a832a0db58813fb6c7babf4dddabcf3426d55a..ef576c382ba5f59490d59fe3e6c003d88858e616

**Review by derrod — APPROVED** [review]

File: buildspec.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (tytan652) [reply buildspec.json:41]: Not resolved since it's still linux-`arm64` and not `linux-aarch64`.

[the remark this answers is not in the document]
Comment (RytoEX) [reply buildspec.json:41]: Ah, you meant the arch indicator. I misread and though you meant the comma, which caused a CI failure. Will fix.

[the remark this answers is not in the document]
Comment (RytoEX) [reply buildspec.json]: Fixed now.

**Pull request merged**